app/scripts/PlotTypeChooser.js

```diff
@@ -91,7 +91,13 @@ class PlotTypeChooser extends React.Component {
     }
 
     return (<div>
-      { AVAILABLE_TRACK_TYPESList }
+      { AVAILABLE_TRACK_TYPESList.length ?
```

Comment:
Same as above instead of `?` one can use `&&` to not having to deal with `: null`. Again this is not important, just FYI

Reply:
Here it works.